doc/README.md

```diff
@@ -0,0 +1,46 @@
+## Getting Started 
+
+[Starting Guide](STARTING_GUIDE.md)
+
+[IDE Setup](IDE_SETUP.md)
+
+
+
+## Development Process
+
+[Coding guidelines](new-guidelines.md)
+
+[Release Checklist](RELEASE_CHECKLIST.md)
+
+[Release Guide](RELEASE_GUIDE.md)
+
+[Merging PR process](merging-pr-process.md)
+
+[Working on PR together with QA team](pipeline_process.md)
+
+
+
+### Testing
```

Comment:
why not `##`?

Reply:
No real reason, just personal preference) `##` looks to huge for me for subsections, but I agree, that probably gradual change is better)

Reply:
ah, no, that was just a mistake)